### ISSUE TITLE
cmake: remove deprecation warnings

### ISSUE THREAD

**Issue by dilyanpalauzov** [libical/libical#299]
cmake 3.8.1 emits:
```
CMake Deprecation Warning at /usr/local/share/cmake-3.8/Modules/FeatureSummary.cmake:638 (message):
   SET_PACKAGE_INFO is deprecated.  Use SET_PACKAGE_PROPERTIES instead.
 Call Stack (most recent call first):
   cmake/modules/FindBDB.cmake:8 (set_package_info)
   CMakeLists.txt:104 (find_package)

 CMake Deprecation Warning at /usr/local/share/cmake-3.8/Modules/FeatureSummary.cmake:638 (message):
   SET_PACKAGE_INFO is deprecated.  Use SET_PACKAGE_PROPERTIES instead.
 Call Stack (most recent call first):
   cmake/modules/FindICU.cmake:17 (set_package_info)
   CMakeLists.txt:137 (find_package)
```

**Comment by winterz** [comment]
confirmed.
unfortunately there is no way to remove those messages that I can see.  well, unless we require CMake 3.8 or higher, which we aren't going to do until a long time from now.

**Comment by winterz** [comment]
fixed in commit 2a5d9b0b63863d036ee2519f69a4a917f7cfbdd7
